python/HeaderService/hslib_salobj.py

```diff
@@ -873,35 +873,6 @@ def collect_from_HeaderService(self, imageName):
         if self.tstand:
             metadata['TSTAND'] = self.tstand
 
-        # If not LATISS stop here
-        if self.config.instrument != 'LATISS':
-            # Update the imageName metadata with new dict
-            self.metadata[imageName].update(metadata)
-            return
-
-        # ---------------------------------------------------------------------
-        # This functions will only be used for LATISS
-
-        if set(('RA', 'DEC', 'ROTPA', 'RADESYS')).issubset(metadata):
-            sensor = self.sensors[0]
-            self.log.info(f"Computing WCS-TAN from RA/DEC/ROTPA for {sensor}")
-            wcs_TAN = self.HDR[imageName].CCDInfo[sensor].wcs_TAN(metadata['RA'],
```

Comment:
You can remove the `wcs_TAN` method from `camera_coords.py` now, as I believe it's unused.

Reply:
Thanks, I've removed the the `wcs_TAN` method from `camera_coords.py`